installation-manager/src/main/resources/org/wildfly/core/instmgr/cli/command_resources.properties

```diff
@@ -13,177 +13,178 @@
+  ensures that the new session uses the most recent changes available on the local installation.
 
 installer.revert.option.repositories.description=\
-  A collection of Maven repositories that contains the artifacts required to prepare an installation at the state specified by 'revision' argument. Multiple repositories are separated by comma. \
-  Valid values are either URLs or ID::URL pairs. This argument allows you to specify an alternative list of repositories for this revert command, \
-  replacing the repositories defined in the subscribed channels.
+  Collection of Maven repositories that contains the artifacts required to prepare an installation at the state specified by the 'revision' option. Specify the \
+  location of the repositories as URLs or ID::URL pairs. You can also specify multiple repositories separated by commas.
 
 installer.revert.option.local-cache.description=\
-  Path to the local Maven repository cache. It overrides the default Maven repository at ~/.m2/repository. The 'path' value represents the location within the remote machine's file system to which this session is connected.
+  Path to the local Maven repository cache. It overrides the default Maven repository at ~/.m2/repository. The 'path' value represents the location on the file \
+  system of the remote machine to which the JBoss CLI is connected.
 
 installer.revert.option.no-resolve-local-cache.description=\
-  Perform the operation without resolving or installing artifacts from/into local maven cache.
+  Perform the operation without resolving or installing artifacts from/into the local maven cache.
 
 installer.revert.option.offline.description=\
   Perform installation from local or file-system Maven repositories only.
 
 installer.revert.option.maven-repo-file.description=\
-  The location of the local Maven Repository Zip file, containing the necessary artifacts to prepare the server installation we want to apply.
+  The location of the local Maven repository zip File. The file contains the necessary artifacts to prepare the server candidate we want to apply.
 
 installer.revert.option.revision.description=\
-  Hash of an installation state of the previous version you want to revert to.
+  Hash of an installation state of the previous version. The base server will be reverted to this installation state.
 
 installer.revert.option.headers.description=\
-  Any Operation headers. Depending on the target environment, preparing a server could take some time because all required artifacts needs to be downloaded \
-  to provision the candidate installation. You can use Operation headers to increase the default blocking timeout which is 300 seconds.
+  Any operation headers. Depending on the target environment, preparing a server could take time because all the required artifacts need to be downloaded to provision \
+  the server candidate. You can use the operation headers to increase the default blocking timeout which is 300 seconds.
```

Comment:
The use of a header as an option is already available for other commands https://docs.wildfly.org/28/Admin_Guide.html#controlling-operation-blocking-time

However, for the installer it is only relevant to supply two options: a blocking-timeout  or a different user role:
--blocking-timeout=<timeout> (it will translate to 
--run-as=<role>

I'm fine if we replace the header here with the above two. I agree with Brian it is more intuitive and less obscure. @jfdenise WDYT?

Reply:
To avoid more last minute changes and since we already have other operations that use an attribute to set any operation Header, we left this operation as it is.